### ISSUE TITLE
SnapKey stopped working on my system

### ISSUE THREAD

**Issue by dedfedorpedor** [cafali/SnapKey#89]
Today I discovered a problem that when I launch SnapKey, my WASD keys stop functioning completely, 
everything works without SnapKey, however, after it is launched, the WASD keys stop functioning

**Comment by cafali** [comment]
Have you checked the provided documentation on how to properly identify and rebind keys in SnapKey?

- [Rebinding Keys](https://github.com/cafali/SnapKey/wiki/Rebinding-Keys)
- [Troubleshooting](https://github.com/cafali/SnapKey/wiki/Troubleshoot)

1. Did this problem start only "today"- so it was fine before?
2. What version of SnapKey are you using?
3. What happens when you run the software as "Administrator"?
4. What keyboard layout are you using? - does it match the default WASD preset in your current config file? 

more info needed....

**Comment by Nakolen1** [comment]
Hello, I have exactly the same problem, I tried to open snapkey in different ways and it still doesn’t work, I’m using the latest version of the application. Could this be related to the fact that I'm using it for CS2?

**Comment by JewelerRr6553** [comment]
Проблема появилась сегодня, ничего не работает, даже если поставить другие клавиши.

**Comment by JewelerRr6553** [comment]
Все работало, сегодня перестал работать, клавиши не работают вообще, не только в игре, а вообще если включён snaptap, если перебиндить, то будут не работать и другие клавиши, если выключить, то все работает

**Comment by ZeNiu85** [comment]
Путин заблокировал всем доступ.
Slava Ukraini!

**Comment by JewelerRr6553** [comment]
после обновления 1.25 все равно не работает

**Comment by minteeaa** [comment]
> Hello, I have exactly the same problem, I tried to open snapkey in different ways and it still doesn’t work, I’m using the latest version of the application. Could this be related to the fact that I'm using it for CS2?
> 
> 

yes. Valve has officially disallowed Snap Tap and alternatives, and is actively adding measures to prevent usage of it in CS2.

do not use SnapKey in CS2.

**Comment by cafali** [comment]
> 1.The problem started today, before all ok 
>2.Latest 1.2.4 
>3.Nothing changes 
>4. WASD 
>Rebind doesnt work UPD:The application stopped working in principle, because when it is turned on, the WASD keys cannot even print text

Since Snapkey was working fine before and then suddenly stopped, please check for any changes made to your computer’s drivers, updates, or other configurations.

Additionally, "WASD" is not a recognized keyboard layout (examples of valid keyboard layouts can be found in the links I previously provided).

Due to a lack of detailed information and the fact that Snapkey is functioning correctly for everyone else, I will move this issue to the discussion section. I am unable to replicate the problem on my system, and no instructions were provided on how to reproduce this behavior or explain what might have caused Snapkey's functionality to break.
